src/app.ts

```diff
@@ -88,8 +88,10 @@ async function app() {
 
   // Load custom resources specified in ig-data folder
   loadCustomResources(path.join(input, 'ig-data', 'input'), defs);
-  if (isIgPubContext && !fs.existsSync(path.join(input, 'ig-data'))) {
+  if (isLegacyIgPubContext && !fs.existsSync(path.join(input, 'ig-data'))) {
     loadCustomResources(path.join(input, '..', 'input'), defs);
+  } else if (isIgPubContext) {
+    loadCustomResources(path.join(input, '..'), defs);
   }
```

Comment:
I'm not sure it makes sense to have three possible ways to load here. The `if` statement on line 91 was originally added to load resources that were in the `<root>/input` folder for the new IG Publisher integration. But now I think that `if` statement is not necessary, since this case is handled by line 93-94.

If someone is using the "Legacy" context, then my expectation is that they would not anticipate that we would load in definitions from `<root>/input`, since that is not a pattern that has traditionally been a part of the "Legacy" context. So would this be better off as just two cases, If we are in "Legacy" mode, load from `ig-data/input`, and if we are in the new mode, load from `<root>/input`.

Let me know if you think I'm off base, I could just be confusing myself about something you've already thought through.

Reply:
Oh, yeah.  I missed this -- but agree w/ @ngfreiter.  In legacy mode, we read _only_ from `ig-data/input`.  In new (non-legacy) mode, we read _only_ from `$root/input`.

Reply:
Ah no, that is a good point. I had lost track of the fact that the load from `<root>/input` case had been added as part of this new configuration, so I can consolidate.

Reply:
I believe 7f06748 should have the simplified configuration, but let me know what you think.